src/bbs2gh/Services/BbsSshArchiveDownloader.cs

```diff
@@ -91,7 +91,14 @@ public async Task<string> Download(long exportJobId, string targetDirectory = IB
 
         if (!_sftpClient.Exists(sourceExportArchiveFullPath))
         {
-            throw new OctoshiftCliException($"Source export archive ({sourceExportArchiveFullPath}) does not exist.");
+            throw new OctoshiftCliException(
+                $"Source export archive ({sourceExportArchiveFullPath}) does not exist." +
+                (BbsSharedHomeDirectory is BbsSettings.DEFAULT_BBS_SHARED_HOME_DIRECTORY_LINUX
+                    ? " This usually happens because the archive could not be found at the default Bitbucket's shared home " +
```

Comment:
Can we change this to?:

> This most likely means that your Bitbucket Server uses a non-default Bitbucket shared home directory, so we couldn't find your archive. You can point the CLI to a non-default shared directory by specifying the --bbs-shared-home option.

Reply:
addressed in bf5a5d6dea62301209ea978394089a368aca2ac6